create_aws_application

```diff
@@ -0,0 +1,71 @@
+#!/bin/sh
+# this is the name of the application you want to create
+NAME=$1
+# these are all the environments. the buckets will be named NAME-ENVIRONMENT
+ENVIRONMENTS=(master develop testing)
+# region where you want to create the bucket
+REGION=eu-central-1
+# aws-cli profile to use (default default)
+PROFILE=${2:-default}
+# policy name (default UserBucketFullControl)
+USERS_POLICY_NAME=${3:-UserBucketFullControl}
```

Comment:
1) You can pass the policy name as a parameter, I do not want to make this script Renuo-specific but available publicly.
2) The advantage is that this script takes care also of that. If the Policy exists very good, otherwise is generated. It wouldn't work otherwise.